sources/packages/backend/libs/sims-db/src/entities/application.model.ts

```diff
@@ -327,6 +328,20 @@ export class Application extends RecordDataModel {
     },
   )
   studentScholasticStandings?: StudentScholasticStanding[];
+  /**
+   * Application offering change request associated with this application.
+   */
+  @OneToMany(
+    () => ApplicationOfferingChangeRequest,
+    (applicationOfferingChangeRequest) =>
+      applicationOfferingChangeRequest.application,
+    {
+      eager: false,
+      cascade: false,
+      nullable: true,
+    },
+  )
+  applicationOfferingChangeRequest?: ApplicationOfferingChangeRequest[];
```

Comment:
@ann-aot @andrewsignori-aot @guru-aot  I am confused here. I was expecting the relationship to be established only between student_assessment and application_offering_change similar to the student appeal. 

Just trying to understand what could be benefit of establishing this one to many relationship?

Reply:
The application can have as many offering change requests, so this is a direct relationship to the change requests from the institution. Its more like a starting point.
But in the student appeal, the student assessment is connected when the ministry approves the appeal request.

Reply:
If I got your question correctly @dheepak-aot , we have been following iOneToMany` in a lot of places wherever needed to make the query easier. I believe we have also used it in the student appeal that you mentioned, 
![image](https://github.com/bcgov/SIMS/assets/77353155/22a30c8c-b98f-4dd8-83bd-02ed68bead72)

Reply:
I understood this from the context of https://github.com/bcgov/SIMS/pull/2013#discussion_r1234459532